### DISCUSSION TITLE
BIRT Report text in ALL CAPS

### DISCUSSION THREAD

**Discussion by tnation1424** [eclipse-birt/birt#1507] [Q&A]
I inherited this report where all the output Field strings values are in UPPERCASE. It's like this through out the report and I want to fix it where is it not always in upper case.  I search the report and I can't locate the setting that is doing this. As a test I inserted a new detail roll and added a new data item that is just a simple text string and prints in all upper case as well.  Is there a "master property" that does this.  I never seen this happen before in other BIRT reports.  I'm sure its something I've overlooked.
![image](https://github.com/eclipse-birt/birt/assets/149100941/d54c2058-f6e7-4710-831c-73c4950e6566)

**Comment by speckyspooky** [comment]
There is a way. You can add on report level a master-style (like CSS) and there you can define for the whole report the text-transformation to UPPERCASE. Al elements of the report like the body and also all master page elements like header and footer will be changed.

You can start with the configuration like:
- go with your mouse on a free empty space on your report to that no element is marked
- Open the context menu with the right click
- meu path: Style => New Style => Predefinded Style => Report


![UPPERCASE](https://github.com/eclipse-birt/birt/assets/41593722/abf895ce-bb9f-4a12-abfd-d59901bec21f)
